xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -975,10 +996,18 @@
         <Parameter Name="values" Type="Microsoft.Extensions.Primitives.StringValues" />
       </Parameters>
       <Docs>
-        <param name="values">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="values">A <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to implicitly convert.</param>
+        <summary>Defines an implicit conversion of a given <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to a string, with multiple values joined as a comma separated string.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Returns `null` where <xref:Microsoft.Extensions.Primitives.StringValues> has been initialized from an empty string array or is <xref:Microsoft.Extensions.Primitives.StringValues.Empty>.
+
+          ]]></format>
+        </remarks>
```

Comment:
This information should not be in remarks, it should be in the returns section. Can you add a proper returns description, including this info?
```suggestion
        <remarks>To be added.</remarks>
```

Reply:
- [ ] needs a proper returns description including this info